pom.xml

```diff
@@ -622,8 +633,21 @@
                         <inputEncoding>UTF-8</inputEncoding>
                         <outputEncoding>UTF-8</outputEncoding>
                     </configuration>
+                    <dependencies>
```

Comment:
Can you tell me what is the purpose of this dependency and why should it be added?

Reply:
@wolfboys Upgrade checkstyle version does not affect the normal build process, but if we not, execute mvn build will print warning like this: ```Old version of checkstyle detected. Consider updating to >= v8.30```